app/services/user_silencer.rb

```diff
@@ -57,8 +57,11 @@ def silence
           user_history: @user_history,
           post_id: @opts[:post_id],
           silenced_till: @user.silenced_till,
-          silenced_at: DateTime.now
+          silenced_at: DateTime.now,
+          silence_message_params: silence_message_params
```

Comment:
Let's add a test for this param as well to demonstrate the intended usage :)

Reply:
Oh right sorry aobut that - just added!